File: tests/class/in-array.swift
public class Size {
	var width: Double
	var height: Double
	public init(width w: Double, height h: Double) {
		width = w
		height = h
	}
}

public func makeSizes(w1: Double, h1: Double, w2: Double, h2: Double) -> [Size] {
	return [Size(width: w1, height: h1), Size(width: w2, height: h2)]
}

public func countSizes(sizes: [Size]) -> Int {
	return sizes.count
}

public func copySizes(sizes: [Size]) -> [Size] {
	return sizes
}

File: tests/class/in-array.js
export function makeSizes$w1$h1$w2$h2$(w1, h1, w2, h2) {
  const size = new Size();
  size.width = w1;
  size.height = h1;
  const size0 = new Size();
  size0.width = w2;
  size0.height = h2;
  return [size, size0];
}
export function countSizes$sizes$(sizes) {
  return sizes.length;
}
export function copySizes$sizes$(sizes) {
  return sizes.slice();
}
export class Size {}